base/show.jl

```diff
@@ -978,22 +983,28 @@ end
 const undef_ref_str = "#undef"
 const undef_ref_alignment = (3,3)
 
+# alignment(X, rows, cols, cols_if_complete, cols_otherwise, sep) returns the
+# alignment for specified parts of array X, returning the (left,right) info.
+# It will look in X's rows, cols (index lists)
+# and figure out what's needed to be fully aligned, for example looking all
+# the way down a column and finding out the maximum size of each element.
+# Parameter sep is number of spaces to put between elements.
 function alignment(
```

Comment:
The additional documentation is great, but you might as well put it in the form of a docstring:
``jl
"""
    alignment(X, rows, cols, cols_if_complete, cols_otherwise, sep)

Returns the alignment for the specified parts of array `X`, returning ....
...
"""
function alignment( ...
```